### PR TITLE
Add crypto to "Le Journal" and little clear code

### PR DESCRIPTION
**Pull request by Vulcainos**
Change departements to new file, more clear !
gitigniore with node.
Clear code in index.
Add Crypto to journal and add api call.
Change UTC time in check loop

### CODE REVIEW

File: discordbot/index.js [hunk 1]
```diff
@@ -66,31 +95,39 @@ setInterval(() => {
 
 // When the client is ready, run this code (only once)
 client.once('ready', () => {
-    getValueOfCurrenciesFrom("EUR");
-    client.user.setActivity('les actus, météo et autres', {type:"WATCHING", url:"https://oxey405.com/projects/monabot"});
+    setValueOfCurrenciesFrom("EUR");
+
+    // Set the client user's activity
+    client.user.setActivity('les actus, météo et autres', { type: "WATCHING"});
```
Comment (Oxey405) [inline discordbot/index.js:101]: Why deleting the URL ?

File: discordbot/index.js [hunk 2]
```diff
@@ -66,31 +95,39 @@ setInterval(() => {
...
+            //For each user subscribed to the bot, send them daily journal.
             subscribedUsers.users.forEach((user) => {
                sendJournalTo(user);
-
            })
         }
-        if(date.getUTCHours() > 19) {
+
+        //Set bot status every day from 21:00 to 06:00 idle
+        if(date.getUTCHours() > 21 || date.getUTCHours() <= 6) {
```
Comment (Oxey405) [inline discordbot/index.js:122]: Ok but why changing the hours ?

**Review by Oxey405 — COMMENTED** [review]
Great PR overall ! There are some minors details to fix before I merge and when those are fixed I'll merge!

File: discordbot/index.js (hunk 1 above)
Comment (Vulcainos) [reply discordbot/index.js:101]: URL is only use if the activity is being streamed, a link to the stream.
https://discord.js.org/#/docs/main/stable/class/Activity?scrollTo=url

Comment (Oxey405) [reply discordbot/index.js:101]: OK

**Review by Oxey405 — APPROVED** [review]
Good for merge

**Pull request merged**